vaas/vaas/router/migrations/0003_redirect_redirectassertion.py

```diff
@@ -18,10 +18,13 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('condition', models.CharField(max_length=512)),
-                ('destination', models.CharField(max_length=512, validators=[django.core.validators.RegexValidator(message='Destination should be relative', regex='^/.*')])),
```

Comment:
We want to delete this `validators=[django.core.validators.RegexValidator(message='Destination should be relative', regex='^/.*')]` ? :) I'm not sure why we deleting this.

Reply:
@jtaczanowski Yes, we removed this validator when we added the ability to redirect to another domain. In this case, relative path validator doesn't make sense. It was removed in this migration https://github.com/allegro/vaas/pull/458/files#diff-495c424d2e337de376b6bf8869cfd25d1fd1d8718428ec23150a2a22854dfb4cL21